src/Steps/Release/PublishRelease.php

```diff
@@ -37,11 +37,33 @@ public function run(InputInterface $input, OutputInterface $output)
     {
         $this->log($output, "Running release of all modules");
 
+        // Release silverstripe/developer-docs before all other modules so that the github workflow
+        // to deploy the updated changelog to netlify is run before CI gets bogged down by everything else.
+        $developerDocs = $this->findLibraryRecursive($this->getReleasePlan(), 'silverstripe/developer-docs');
+        $this->releaseLibrary($output, $developerDocs, true);
+
         $this->releaseRecursive($output, $this->getReleasePlan());
 
         $this->log($output, "All releases published");
     }
 
+    /**
+     * Find a library in the release plan
+     */
+    protected function findLibraryRecursive(LibraryRelease $releasePlanNode, string $name): ?LibraryRelease
```

Comment:
I used protected instead of private because everything else in the file uses protected